src/main/sensors/gyroanalyse.c

```diff
@@ -132,6 +122,12 @@ const gyroFftData_t *gyroFftData(int axis)
  */
 void gyroDataAnalyse(const gyroDev_t *gyroDev, biquadFilter_t *notchFilterDyn)
 {
+    // accumulator for oversampled data => no aliasing and less noise
+    static FAST_RAM float fftAcc[XYZ_AXIS_COUNT] = {0, 0, 0};
```

Comment:
Zero-initialized by default as well.
Making these variables function-local static has no benefit though, therefore I suggest moving them back to global scope.

Reply:
No functional change but it is used only in this method so I limited visibility scope of the varibale.

Reply:
Function-local statics beat additional semantics for ensuring initialization (not for zeo-initialization though). It's better not to use them without real necessity

Reply:
So should I put them back?

Reply:
No big deal, if you want locality leave them here, I'll review the generated code to ensure it doesn't imposr any additional overhead

Reply:
ok, thank you.